### ISSUE TITLE
[Bug] The Example of MoNet is not Reproducible

### ISSUE THREAD

**Issue by chwan1016** [dmlc/dgl#6100]
## 🐛 Bug

I was running the example of MoNet (https://github.com/dmlc/dgl/tree/master/examples/pytorch/monet). The final accuracy for Cora is typically between 74%~78%. Validation accuracy never exceeds 80%. The expected accuracy is ~81.6%. I'm not sure if this is caused by the upgrade of the backend framework.

## Environment

 - DGL Version (e.g., 1.0): DGL 1.1
 - Backend Library & Version (e.g., PyTorch 0.4.1, MXNet/Gluon 1.3): PyTorch 2.1.0
 - OS (e.g., Linux): Linux

**Comment by frozenbugs** [comment]
@chwan1016 thanks for catching it, cora is a pretty small graph so it is very sensitive to small changes, this example is an example that demonstrate the usability of DGL so we did not keep an eye on the acc of the model.

Do you happen to know why the accuracy is dropped? If not, feel free to change the README to Cora: ~0.75.
